### ISSUE TITLE
Can I use lr_scheduler of Pytorch in model.fit ?

### ISSUE THREAD

**Issue by kyuchoi** [havakv/pycox#103]
First of all, thank you for your great works !!

Can I use lr_scheduler of Pytorch in model.fit ?
If it's possible, then how can I modify the following code to use the scheduler for learning rate?

log = model.fit(x_train, y_train, batch_size, epochs, callbacks, val_data=val)

Many thanks
